### PR TITLE
Load real allowance data

### PR DESCRIPTION
**Pull request by bumi**
ToDo:

- [ ] remove search bar?
- [ ] show more data
- [ ] link to detail page
- [ ] rename `percentag` to `percentageRemaining`
- [ ] relevant badges

### CODE REVIEW

File: src/app/components/PublishersTable.jsx [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
 
 export default function PublishersTable({ publishers }) {
+  const deletePublisher = (id) => {
+    utils.call("deleteAllowance", { id });
```
Comment (bumi) [inline src/app/components/PublishersTable.jsx:10]: @dylancom this is the delete call. how do we best handle the refresh of the publishers/or the removal of that specific publisher?

File: src/app/components/PublishersTable.jsx [hunk 2]
```diff
@@ -29,30 +37,33 @@ export default function PublishersTable({ publishers }) {
...
-                    <div
-                      style={{ width: `${publisher.percentage}%` }}
-                      className="ml-auto h-2 shadow-none rounded flex flex-col text-center whitespace-nowrap text-white justify-center bg-blue-500"
-                    />
-                    <span
-                      className={`ml-2 text-sm font-semibold text-blue-500`}
-                    >
-                      {publisher.percentage}%
-                    </span>
+              <span onClick={deletePublisher(publisher.id)}>delete</span>
```
Comment (dylancom) [inline src/app/components/PublishersTable.jsx:46]: Should be () => deletePublisher(publisher.id). Now the function is invoked directly.

File: src/app/components/PublishersTable.jsx (hunk 1 above)
Comment (dylancom) [reply src/app/components/PublishersTable.jsx:10]: If you move the deletePublisher function to the Publishers screen, you can directly update the state from there.
Inside the deletePublisher function you can add: setData(data.filter(publisher => publisher.id !== id)), which will refresh the publishers table.
From Publishers.tsx you need to pass down deletePublisher as a prop to PublishersTable.

This also makes sure that your component stays "dumb" and "smart" things are handled within your screens (smart components). (Clean code)

File: src/app/components/PublishersTable.jsx [hunk 3]
```diff
@@ -1,8 +1,16 @@
...
 
 export default function PublishersTable({ publishers }) {
```
Comment (dylancom) [inline src/app/components/PublishersTable.jsx:8]: Add extra prop: deletePublisher (or handleDelete for e.g. how you prefer).

File: src/app/components/PublishersTable.jsx (hunk 2 above)
Comment (bumi) [reply src/app/components/PublishersTable.jsx:46]: 👍 yep. thanks

**Comment by bumi** [comment]
mergeing this to be able to continue with #89

**Pull request merged**